package.json

```diff
@@ -77,6 +77,7 @@
     "now": "^11.0.2",
     "now-cli": "^0.0.0",
     "prettier-eslint": "^8.8.1",
+    "react-test-renderer": "^16.3.2",
```

Comment:
why this? we use enzyme https://github.com/airbnb/enzyme

Reply:
I can replace it with enzime :-)

Reply:
Done!